### PR TITLE
integration tests setup

### PR DESCRIPTION
**Pull request by dhorelik**
sets up the overall django CMS integration tests infrastructure
- [x] setting up casperjs on phantomjs
- [ ] getting the setup to run on travis & locally (in a simple one command to rule them all way)
- [x] ability to run tests separately file by file locally vs all at once on CI
- [x] readme / docs update
- [ ] simple test suite (ex. checking page title)

### CODE REVIEW

File: cms/tests/frontend/unit/cms.base.test.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 describe('cms.base.js', function () {
+    'use strict';
```
Comment (vxsx) [inline cms/tests/frontend/unit/cms.base.test.js]: doesn't `node: true` solve it in the .jshintrc ?

File: npm-shrinkwrap.json [hunk 2]
```diff
@@ -25,6 +25,11 @@
         }
       }
     },
+    "casperjs": {
+      "version": "1.1.0-beta3",
```
Comment (FinalAngel) [inline npm-shrinkwrap.json]: why can't we use the stable version?

File: test_requirements/django-1.9.txt [hunk 3]
```diff
@@ -5,6 +5,5 @@ django-classy-tags>=0.6.1
 djangocms-text-ckeditor==2.7
 django-hvad==1.5
 django-formtools
-django-classy-tags>=0.6.1
```
Comment (FinalAngel) [inline test_requirements/django-1.9.txt]: should that be in here? :D

File: npm-shrinkwrap.json (hunk 2 above)
Comment (dhorelik) [inline npm-shrinkwrap.json]: this is the first release installable via the npm, and it has a number of issues fixed, see the [changelog](https://github.com/n1k0/casperjs/releases/tag/1.1-beta3). The next stable release is going to support phantomjs 2.0, so we'll update to that eventually

File: test_requirements/django-1.9.txt (hunk 3 above)
Comment (vxsx) [inline test_requirements/django-1.9.txt]: yeah it's a duplicate from line 4

Comment (vxsx) [inline test_requirements/django-1.9.txt]: but tbh 1.9 is failing on upstream, i think we can revert for the moment

**Comment by vxsx** [comment]
LGTM

**Pull request merged**